src/sentry/api/serializers/models/rule.py

```diff
@@ -165,6 +165,18 @@ def get_attrs(self, item_list, user, **kwargs):
             for rule in item_list:
                 result[rule]["last_triggered"] = last_triggered_lookup.get(rule.id, None)
 
+        neglected_rule = NeglectedRule.objects.filter(
+            rule__in=item_list,
+            opted_out=False,
+            sent_initial_email_date__isnull=False,
+        )
+        rule_snooze = RuleSnooze.objects.filter(
+            Q(user_id=user.id) | Q(user_id=None), rule__in=[item.id for item in item_list]
+        )
+        for rule in item_list:
+            result[rule]["neglected_rule"] = neglected_rule
+            result[rule]["snooze"] = rule_snooze
```

Comment:
You need to fetch the neglected rule and snooze that is specific to the `rule`. Usually we build a lookup table to do this, see `last_triggered_lookup` for an example

Reply:
ok hopefully I did it right this time